src/std/ucs2.c

```diff
@@ -101,7 +102,8 @@ int utoi( const uchar *str, uchar **end ) {
 	}
 	buf[i] = 0;
 	result = strtol(buf,&bend,10);
-	*end = str + (bend - buf);
+	uchar *strcpy = ustrdup(str);
```

Comment:
Here as well :)

Reply:
Yes, I saw it just a bit after hitting Sync. Fixed. ^.^`